test/SourceKit/CodeComplete/complete_sort_order.swift

```diff
@@ -77,7 +77,7 @@ func test5() {
 // STMT_1-LABEL: Results for filterText: ret [
 // STMT_1-NEXT:    return
 // STMT_1-NEXT:    retLocal
-// STMT_1-NEXT:    repeat
+// STMT_1:         repeat
```

Comment:
This is because `withTaskGroup(of:returning:body:)` and `withThrowingTaskGroup(of:returning:body:)` are ranked above `repeat`. They used to be `InvalidAsyncContext`.